.nuget/create_nuget.py

```diff
@@ -64,7 +64,7 @@
     </ItemGroup>
     </Project>
     """
-    target_str = r"""
+    target_str = f"""
```

Comment:
```suggestion
    target_str = r"""
```

This needs to be a raw string and does not rely on any Python template. Please change this back.